Kitodo/src/test/java/org/kitodo/production/interfaces/activemq/KitodoScriptProcessorIT.java

```diff
@@ -0,0 +1,440 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+package org.kitodo.production.interfaces.activemq;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+
+import javax.jms.Destination;
+import javax.jms.JMSException;
+import javax.jms.MapMessage;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.kitodo.MockDatabase;
+import org.kitodo.SecurityTestUtils;
+import org.kitodo.data.database.beans.Process;
+import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.command.KitodoScriptService;
+
+public class KitodoScriptProcessorIT {
+    @BeforeAll
```

Comment:
This should be executed before any test is executed or?
```suggestion
    @BeforeEach
```

Reply:
It doesn't matter here, as there is only one test included. As far as I can see, the database is only created once for all other test classes, not for each test. That's why I would do the same here.

Reply:
With `BeforeEach` and `AfterEach` would ensure, that every test is executed in the same conditions. Okay, there is currently only one test in this test class but it must / should be changed on adding an other test where I think this would got forgotten.